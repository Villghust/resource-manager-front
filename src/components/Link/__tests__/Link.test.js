import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import { Link } from '../index';

it('renders correctly', () => {
    const tree = renderer
        .create(
            <BrowserRouter>
                <Link to="/reports">Relat√≥rios</Link>
            </BrowserRouter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
