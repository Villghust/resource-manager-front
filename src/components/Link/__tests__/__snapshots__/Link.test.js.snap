// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="makeStyles-root-1"
  href="/reports"
  onClick={[Function]}
>
  Relat√≥rios
</a>
`;
